--- tab_local.lua.orig	2026-06-03 09:49:44.568241064 +0200
+++ tab_local.lua	2026-06-03 09:49:11.984186409 +0200
@@ -198,6 +198,7 @@
 				dump(core.settings:get_bool("enable_damage")) .. "]"
 			y = y + yo
 		end
+		disabled_settings["enable_server"] = true
 		if disabled_settings["enable_server"] == nil then
 			host = "checkbox[0,"..y..";cb_server;".. fgettext("Host Server") ..";" ..
 				dump(core.settings:get_bool("enable_server")) .. "]"
